src/misc/lv_event.c

```diff
@@ -109,12 +128,12 @@ bool lv_event_remove_dsc(lv_event_list_t * list, lv_event_dsc_t * dsc)
     LV_ASSERT_NULL(list);
     LV_ASSERT_NULL(dsc);
 
-    int size = lv_array_size(list);
-    lv_event_dsc_t ** events = lv_array_front(list);
+    int size = event_array_size(list);
     for(int i = 0; i < size; i++) {
+        lv_event_dsc_t ** events = event_array_at(list, i);
         if(events[i] == dsc) {
```

Comment:
Should this be `if(*events == dsc)` ?
Or we should simply use `events[i]` and keep original `lv_event_dsc_t ** events = event_array_front(list);`


```c
    int size = event_array_size(list);
    lv_event_dsc_t ** events = event_array_front(list);
    for(int i = 0; i < size; i++) {
        if(events[i] == dsc) {
            event_mark_deleting(list, events[i]);
            cleanup_event_list(list);
            return true;
        }